src/app/sol-warnings.js

```diff
@@ -0,0 +1,54 @@
+module.exports = { find: find };
+
+// source: just a big string of the current tab's source code.
+//
+// compiledResult: an object with different keys based on success/failure of compilation.
+//   success keys: 'contracts' (contains opcodes, bytecode) and 'sources' (contains AST)
+//   error key: 'error' (contains compile error string)
+
+function find (source, compiledResult) {
+  var messages = [];
+
+  if (compiledResult) {
+    if (compiledResult.contracts) {
+      Object.keys(compiledResult.contracts).forEach(function (contractName) {
+        // @TODO: use compiledResult.contracts[contractName].opcodes, interface, etc
+      });
+    }
+    if (compiledResult.sources) {
+      Object.keys(compiledResult.sources).forEach(function (tabName) {
+        var ast = compiledResult.sources[tabName].AST;
+        if (searchAST(ast, {member_name: 'send'})) {
+          messages.push('Caution: You are using the send method.\nPlease read <a target="_blank" href="http://hackingdistributed.com/2016/06/16/scanning-live-ethereum-contracts-for-bugs/">this</a> before deploying your code.');
```

Comment:
I would rather warn on `call` or `call.value` because that is much more dangerous.